### PR TITLE
making styles as important

### LINKED ISSUE #11645: !important amp-story system styling
Many styles within the `amp-story` component are not intended to be overwritten, but can be accidentally overwritten depending on publisher styles.  We should `!important` all styles that are not intended to be overwritten.

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #11645

- Adding important to `amp-story.css`

I have made some changes to amp-story.css assuming that user must not be able to change these. let me know if we need to have `important` on some selected styles

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

**Comment by prateekbh** [comment]
work in progress

File: extensions/amp-story/0.1/amp-story.css [hunk 1]
```diff
@@ -266,7 +266,7 @@ amp-story-grid-layer {
   position: absolute !important;
   right: 0 !important;
   top: 0 !important;
-  padding: 68px 32px 32px; /* 44 + 24 */
+  padding: 68px 32px 32px !important; /* 44 + 24 */
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 2]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
 }
 
 .i-amphtml-story-grid-template-vertical {
-  align-content: start;
+  align-content: start !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 3]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
   grid-auto-flow: row !important;
-  grid-gap: 16px;
+  grid-gap: 16px !important;
   grid-template-columns: 100% !important;
-  justify-content: stretch;
-  justify-items: start;
-  padding: 32px;
+  justify-content: stretch !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 4]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
 }
 
 .i-amphtml-story-grid-template-vertical {
-  align-content: start;
+  align-content: start !important;
   grid-auto-flow: row !important;
-  grid-gap: 16px;
+  grid-gap: 16px !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 5]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
   grid-template-columns: 100% !important;
-  justify-content: stretch;
-  justify-items: start;
-  padding: 32px;
+  justify-content: stretch !important;
+  justify-items: start !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 6]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
+  padding: 32px !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 7]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
 }
 
 .i-amphtml-story-grid-template-vertical > * {
-  width: 100%;
+  width: 100% !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 8]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
 }
 
 .i-amphtml-story-grid-template-horizontal {
-  align-content: stretch;
-  align-items: start;
+  align-content: stretch !important;
+  align-items: start !important;
   grid-auto-flow: column !important;
-  grid-gap: 16px;
+  grid-gap: 16px !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 9]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
   grid-template-rows: 100% !important;
-  justify-content: start;
-  padding: 32px;
+  justify-content: start !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 10]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
+  padding: 32px !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 11]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
 .i-amphtml-story-grid-template-vertical > * {
-  width: 100%;
+  width: 100% !important;
 }
 
 .i-amphtml-story-grid-template-horizontal {
-  align-content: stretch;
-  align-items: start;
+  align-content: stretch !important;
+  align-items: start !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 12]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
 
 .i-amphtml-story-grid-template-vertical > * {
-  width: 100%;
+  width: 100% !important;
 }
 
 .i-amphtml-story-grid-template-horizontal {
-  align-content: stretch;
-  align-items: start;
+  align-content: stretch !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 13]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
+  padding: 32px !important;
 }
 
 .i-amphtml-story-grid-template-thirds {
   grid-template-rows: repeat(3, 1fr) !important;
   grid-template-areas: "upper-third"
                        "middle-third"
                        "lower-third" !important;
-  padding: 32px;
+  padding: 32px !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: Can be overridden

File: extensions/amp-story/0.1/amp-story.css [hunk 14]
```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
...
 
 .i-amphtml-story-bookend-article-heading {
-  font-family: 'Roboto', sans-serif;
-  font-weight: 300;
-  font-size: 16px;
-  color: #ffff;
+  font-family: 'Roboto', sans-serif !important;
+  font-weight: 300 !important;
+  font-size: 16px !important;
+  color: #ffff !important;
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story.css]: nit: can you also drop the extra `f` from this line?

File: extensions/amp-story/0.1/amp-story.css (hunk 1 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 2 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 4 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 3 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 6 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 5 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 7 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 12 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 11 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 8 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 9 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 10 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 13 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story.css (hunk 14 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.css]: Done

File: extensions/amp-story/0.1/amp-story-desktop-user-overridable.css [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+
+[desktop] amp-story-page {
+  border-radius: 3px;
+}
+
+[desktop] amp-story-page {
+  max-height: 736px;
+  max-width: 338px;
+  box-shadow: 0 0 15px rgba(0, 0, 0, .4);
+}
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story-desktop-user-overridable.css]: nit: newline at end of file

File: extensions/amp-story/0.1/amp-story-desktop-user-overridable.css [hunk 16]
```diff
@@ -0,0 +1,13 @@
+.i-amphtml-story-background {
+  background-color: transparent;
+}
+
+[desktop] amp-story-page {
```
Comment (newmuis) [inline extensions/amp-story/0.1/amp-story-desktop-user-overridable.css]: I don't actually think the current plan is to let any of the things from here down be overridden

**Comment by newmuis** [comment]
Tests failed; looks like it's only because the new files are missing the licensing boilerplate.

File: extensions/amp-story/0.1/amp-story-desktop-user-overridable.css (hunk 16 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story-desktop-user-overridable.css]: Done

File: extensions/amp-story/0.1/amp-story-desktop-user-overridable.css (hunk 15 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story-desktop-user-overridable.css]: Done

**Pull request merged**